src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -76,6 +79,25 @@ public class OSIsoftServer {
   /** Constructs the OSIsoftServer object. */
   public OSIsoftServer() {}
 
+  /** HTTP Method strings */
+  static final String HTTP_METHOD_POST = "POST";
+
+  static final String HTTP_METHOD_GET = "GET";
+
+  /**
+   * Repeated Exception handling code for HTTP requests
+   *
+   * @param e Exception
+   * @param ErrorMessage Specific message to log
+   * @return error code
+   */
+  private static int RequestHTTPSError(Exception e, String ErrorMessage) {

```

Comment:
See comment about `res` variable in the `RequestHTTPS` method. It is redundant, and this method should simply be a `void` (return nothing). 

If the code is calling this method, it already knows there is an error, so its redundant to `return SCHttpUtility.HTTPX_CODE_EWON_ERROR;`

Reply:
I have not addressed this comment